A32NX/html_ui/Pages/A32NX_Core/A32NX_FWC.js

```diff
@@ -358,6 +348,15 @@ class A32NX_FWC {
             return;
         }
 
+        // Exit when:
+        // - Landing gear down
+        // - Glide slope captured
+        const landingGearIsDown = !SimVar.GetSimVarValue("IS GEAR RETRACTABLE", "Boolean") || SimVar.GetSimVarValue("GEAR HANDLE POSITION", "Boolean");
+        const glideSlopeCaptured = SimVar.GetSimVarValue("L:GLIDE_SLOPE_CAPTURED", "bool") === 1;
+        if (landingGearIsDown || glideSlopeCaptured) {
+            return;
+        }
```

Comment:
Have you checked whether this should also turn off the C Chord? Or does it just inhibit a new one from starting?

If it also stops it, you should probably be unsetting a SimVar here.

Reply:
idk about that. inhibit mean a logic is stop? or sound also stop?

before fwc document, seems the former is make sense i think (i'm not good at read logic because studying now XD.)